### PR TITLE
Report on count of TLS handshake messages.

### PR DESCRIPTION
**Pull request by mgritter**
Depends on https://github.com/akitasoftware/akita-libs/pull/159

Example output:
```
mark@ubuntu:~/akita-cli$ sudo -E bin/akita apidump --project mgg-test 
[INFO] Akita Agent 0.0.0
[INFO] Created new trace on Akita Cloud: akita://mgg-test:trace:lace-pig-d35eeeab
[INFO] Running learn mode on interfaces lo, ens33, docker0, br-a17c933efd5a
[WARNING] --filter flag is not set, this means that all network traffic is treated as your API traffic
[INFO] Send SIGINT (Ctrl-C) to stop...
[INFO] Printing packet capture statistics after 60 seconds of capture.
[INFO] ==================================================
[INFO] Packets per interface:
[INFO]                             TCP       HTTP      TLS      
[INFO]       interface       dir packets   req  resp hello   unk
[INFO]              lo   MATCHED       0     0     0     0   244
[INFO]           ens33   MATCHED   29955    32    30   457 16791
[INFO]         docker0   MATCHED       0     0     0     0     0
[INFO] br-a17c933efd5a   MATCHED       0     0     0     0     0
[INFO] ==================================================
```

### CODE REVIEW

File: apidump/summary.go [hunk 1]
```diff
@@ -87,13 +86,13 @@ func (s *Summary) PrintWarnings() {
 					"This may mean your filter is incorrect, such as the wrong TCP port."
 				printer.Stderr.Infof("%s\n", printer.Color.Yellow(msg))
 			}
+		} else if totalCount.TLSHello > 0 {
+			msg := fmt.Sprintf("Captured %d TLS handshake messages out of %d total TCP seegments. ", totalCount.TLSHello, totalCount.TCPPackets) +
```
Comment (thatplguy) [inline apidump/summary.go:90]:
```suggestion
			msg := fmt.Sprintf("Captured %d TLS handshake messages out of %d total TCP segments. ", totalCount.TLSHello, totalCount.TCPPackets) +
```

**Review by thatplguy — APPROVED** [review]

File: apidump/summary.go [hunk 2]
```diff
@@ -129,45 +128,47 @@ func DumpPacketCounters(logf func(f string, args ...interface{}), interfaces map
...
+	logf("========================================================\n")
 	logf("Packets per port:\n")
-	logf("%8v %7v %11v %5v\n", "", "TCP  ", "HTTP   ", "")
-	logf("%8v %7v %5v %5v %5v\n", "port", "packets", "req", "resp", "unk")
+	logf("%8v %7v %11v %5v %5v\n", "", "TCP  ", "HTTP   ", "TLS  ", "")
+	logf("%8v %7v %5v %5v %5v %5v\n", "port", "packets", "req", "resp", "hello", "unk")
 	for i, summary := range toReport {
 		if filterStates[i] == matchedFilter {
-			logf("--------- matching filter --------\n")
+			logf("--------- matching filter --------------\n")
```
Comment (liujed) [inline apidump/summary.go:157]:
```suggestion
			logf("----------- matching filter ------------\n")
```

File: apidump/summary.go [hunk 3]
```diff
@@ -129,45 +128,47 @@ func DumpPacketCounters(logf func(f string, args ...interface{}), interfaces map
...
 		} else {
-			logf("------- not matching filter ------\n")
+			logf("------- not matching filter ------------\n")
```
Comment (liujed) [inline apidump/summary.go:159]:
```suggestion
			logf("--------- not matching filter ----------\n")
```

**Review by liujed — APPROVED** [review]

**Comment by liujed** [comment]
@mgritter, did you forget to update akita-libs after merging akitasoftware/akita-libs#159?

**Comment by mgritter** [comment]
Yes, dangit, I did.  :(

**Pull request merged**